content/docs/en/elements/components/search-bar.md

```diff
@@ -1,5 +1,28 @@
 ---
 title: SearchBar
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_button_.button
 ---
 
-TODO
+The SearchBar widget provides a user interface for entering search queries and submitting requests to the search provider.
+
+---
+
+```html
+<SearchBar hint="Search hint" :text="searchPhrase" @textChange="onTextChanged($event)" @submit="onSubmit($event)"></SearchBar>
```

Comment:
I would simplify this (passing the $event property is the default behavior and can be omitted)

```html
<SearchBar hint="Search hint" :text="searchPhrase" @textChange="onTextChanged" @submit="onSubmit"/>
```